docs/designers-developers/glossary.md

```diff
@@ -1,21 +1,61 @@
-- **RichText**: A common component enabling rich content editing including bold, italics, hyperlinks, etc. It is not too much unlike the single editor region of the legacy post editor, and is in fact powered by the same TinyMCE library.
-- **Inspector**: A block settings region shown in place of the post settings when a block is selected. Fields may be shown here to allow the user to customize the selected block.
-- **Post settings**: A sidebar region containing metadata fields for the post, including scheduling, visibility, terms, and featured image.
-- **Reusable block**:
-- **Sidebar**:
-- **Serialization**: The process of converting a block's attributes object into HTML markup, typically occurring when saving the post.
-- **Static block**: A type of block where the content of which is known at the time of saving a post. A static block will be saved with HTML markup directly in post content.
-- **TinyMCE**: [TinyMCE](https://www.tinymce.com/) is a web-based JavaScript WYSIWYG (What You See Is What You Get) editor.
-- **Toolbar**: A set of button controls. In the context of a block, usually referring to the toolbar of block controls shown above the selected block.
-- **Template**:
+<dl>
+<dt>Attribute sources</dt>
+<dd>An object describing the attributes shape of a block. The keys can be named as most appropriate to describe the state of a block type. The value for each key is a function which describes the strategy by which the attribute value should be extracted from the content of a saved post's content. When processed, a new object is created, taking the form of the keys defined in the attribute sources, where each value is the result of the attribute source function.</dd>
+
+<dt>Attributes</dt>
+<dd>The object representation of the current state of a block in post content. When loading a saved post, this is determined by the attribute sources for the block type. These values can change over time during an editing session when the user modifies a block, and are used when determining how to serialize the block.</dd>
+
+<dt>Block</dt>
+<dd>The abstract term used to describe units of markup that, composed together, form the content or layout of a webpage. The idea combines concepts of what in WordPress today we achieve with shortcodes, custom HTML, and embed discovery into a single consistent API and user experience.</dd>
+
+<dt>Block Categories</dt>
+<dd>These are not a WordPress taxonomy, but instead used internally to sort blocks in the Block Inserter.</dd>
+
+<dt>Block Inserter</dt>
+<dd>Primary interface for selecting from the available blocks, triggered by plus icon buttons on Blocks or in the top-left of the editor interface.</dd>
+
+<dt>Block name</dt>
+<dd>A unique identifier for a block type, consisting of a plugin-specific namespace and a short label describing the block's intent. e.g. `core/image`</dd>
```

Comment:
Since we had to update the link (in the TinyMCE description) from the Markdown version to HTML, will we have the same problem with the code here?

Reply:
Updated to use <code></code> tags.